packages/kbn-reporting/export_types/png/types.ts

```diff
@@ -1,13 +1,14 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
  */
 
+import type { BaseParams, BasePayload } from '@kbn/reporting-export-types-helpers-server';
+import type { LocatorParams } from '@kbn/reporting-common/url';
 import type { LayoutParams } from '@kbn/screenshotting-plugin/common';
-import type { LocatorParams } from '../url';
-import type { BaseParams, BasePayload } from '../base';
```

Comment:
It seems like these types should be in `packages/kbn-reporting/export_types_helpers_server/types.ts`